searches/bin.cpp

```diff
@@ -0,0 +1,62 @@
+#include<iostream>
+#include<math.h>
+#include<fstream>
+#include<vector>
+#include<sstream>
+#include<algorithm>
+

```

Comment:
Please use this code style conventions
https://google.github.io/styleguide/cppguide.html
https://google.github.io/styleguide/cppguide.html#Formatting